pom.xml

```diff
@@ -2,18 +2,18 @@
 	<modelVersion>4.0.0</modelVersion>
 
 	<groupId>org.sentrysoftware</groupId>
-	<artifactId>MYARTIFACT</artifactId>
-	<name>MY PROJECT</name>
-	<version>0.1.00-SNAPSHOT</version>
-	<description>SOME DESCRIPTION</description>
+	<artifactId>tablejoin</artifactId>
+	<name>TableJoin Client</name>
+	<version>1.0.00-SNAPSHOT</version>
+	<description>TableJoin Client Library for Java</description>
```

Comment:
Update the description to:
`<description>TableJoin Utility for Java</description>`